.github/workflows/ci.yml

```diff
@@ -1,9 +1,7 @@
 name: Tests Pipeline
 
 on:
-  pull_request:
-  push:
-    branches: ['main']
+  pull_request: {}
```

Comment:
I intend to have the CI run on PRs that target the `main` branch since this is the workflow of this repository.

Reply:
Oh, sorry; I did not intend to change that workflow! Fixed.